real_preprocessing/src/corner_detection.cpp

```diff
@@ -0,0 +1,91 @@
+#include "ros/ros.h"
+#include "apriltag_ros/AprilTagDetectionArray.h"
+#include "tf_conversions/tf_eigen.h"
+#include "Eigen/Dense"
+#include "fstream"
+#include "string"
+#include "iostream"
+
+int count(0);
+std::string capture("1");//prevents first iteration YAML dump
+std::ofstream fout;
+
+//returns YAML formatted pixel coordinate string given desired corner, cam_T_tag transform and tag size
+std::string pixCalc(Eigen::MatrixXd cam_T_tag, double tag_size, int x, int y) 
+{
+  //intrinsic specification
+  Eigen::MatrixXd intrinsic(3, 3);  
+  intrinsic << 630.3467672874693, 0, 314.10832937485145, 0, 624.9044125522948, 241.59156511756711, 0, 0, 1;
```

Comment:
We should really unhardcode this and find a way to pass the values into the node.